packages/in_app_purchase/lib/src/store_kit_wrappers/sk_payment_transaction_wrappers.dart

```diff
@@ -22,7 +24,9 @@ abstract class SKTransactionObserverWrapper {
   void removedTransactions({List<SKPaymentTransactionWrapper> transactions});
 
   /// Triggered when there is an error while restoring transactions.
-  void restoreCompletedTransactions({Error error});
+  ///
+  /// The error is represented in a Map. The map contains `errorCode` and `message`
+  void restoreCompletedTransactionsFailed({Map<String, String> error});
```

Comment:
We should probably define a class with these props on it instead of using a map. The Map leaves this open to a bunch of dynamic typing kinds of bugs.